subsystem/intake.py

```diff
@@ -0,0 +1,35 @@
+import wpilib
+import constants
+import config
+from robotpy_toolkit_7407 import Subsystem
+from robotpy_toolkit_7407.motors import SparkMax, SparkMaxConfig
+from robotpy_toolkit_7407.pneumatics.pistons import DoubleSolenoidPiston
+from dataclasses import dataclass
+
+@dataclass
+class Intake(Subsystem):
+    intake_motor: SparkMax
+    intake_piston: DoubleSolenoidPiston
+
+    intake_speed: float
+    intake_active: bool
+
+    def init(self):
+        self.intake_motor = SparkMax(config.intake_motor_id)
```

Comment:
I've got the same comment as [here](https://github.com/Choate-Robotics/7407-DriveCode-ChargedUp/pull/9#discussion_r1083373806) and [here](https://github.com/Choate-Robotics/7407-DriveCode-ChargedUp/pull/13#discussion_r1084620714) about setting up the dependencies outside the class. You're already halfway there by using `@dataclass` though!